index.html

```diff
@@ -40,6 +40,20 @@ <h4 class="sub-title">I'am a software Developer</h4>
     </ul>
   </div>
 </section>  
+<section class="section2">
+  <h2 class="heading">My Recent Works</h2>
+  <hr>
+  <img class="placeholder" src="images/Placeholder.png" alt="placeholder">
+  <h3 class="stories">Multi-Post Stories</h3>
+  <p class="section-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+  <ul class="tags">
+    <a href="#"><li class="tag1">CSS</li></a>
+    <a href="#"><li class="tag2">HTML</li></a>
+    <a href="#"><li class="tag3">Bootstrap</li></a>
+    <a href="#"><li class="tag4">Ruby</li></a>
```

Comment:
- [ ] Same too here, your `a` tags should be placed inside the `li` tags. Also, your technologies here are not aligned and the right styling has not been used. I suggest using a 'text-decoration: none' on the `a` tags to remove the initial blue color before styling. Below is your design and the design from Figma.
- Your design:
![f2](https://user-images.githubusercontent.com/85113567/148491162-1b244101-120e-492a-8e26-87264380aaba.jpg)
- Figma's design:
![f1](https://user-images.githubusercontent.com/85113567/148491194-fece42e7-b207-4f28-97f3-8204ed9d2200.jpg)